debian/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+RUN apt-get update && apt-get install -qqy curl python2.7 gcc python-dev python-setuptools apt-transport-https lsb-release  && \
```

Comment:
@salrashid123 I don't see the packages listed on this line on the page you linked. 

I understand python-dev, gcc is probably required for crcmod, however the `apt-get install google-cloud-sdk` must be specifying what dependencies it needs. Therefore you should not include dependencies that `apt-get install google-cloud-sdk` would fetch here.